api/src/data_inclusion/api/inclusion_data/schemas.py

```diff
@@ -51,4 +56,28 @@ class ServiceSearchResult(BaseModel):
 
 
 class DetailedStructure(Structure):
+    doublons: Annotated[
+        list[schema.Structure],
+        Field(description="Doublons connus de la structure"),
+    ]
     services: list[Service]
+
+
+class SourceIdDict(TypedDict):
```

Comment:
oui possible, mais

```suggestion
class SourceIdDict(BaseModel):
```

est équivalent et donc je suis pour ne pas introduire une nouvelle façon de faire

Reply:
OK :+1: d'autant plus qu'on va avoir besoin de sérialisation avec la suppression de la méthode `serialize_doublons`. Je voulais éviter de créer carrément un modèle pour marquer la différence avec les "vraies" structures, mais tu as raison c'est mieux d'aligner.